tensorrt_llm/serve/cluster_storage.py

```
@@ -1,20 +1,62 @@
+        if dist_file_str.endswith(candidates):
+            return dist.locate_file(dist_file)
+    return None
+
+
+def load_module_from_distribution(dist_name: str, module_name: str):
+    dist = importlib_metadata.distribution(dist_name)
+
+    module_file = _find_module_file_in_distribution(dist, module_name)
+    if not module_file:
+        raise ModuleNotFoundError(
+            f"{module_name} not found in distribution {dist_name}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)